static-builder.Dockerfile

```diff
@@ -57,7 +58,7 @@ RUN git clone --depth=1 https://github.com/crazywhalecc/static-php-cli . && \
 
 RUN --mount=type=secret,id=github-token GITHUB_TOKEN=$(cat /run/secrets/github-token) ./bin/spc download --with-php=$PHP_VERSION --all
 
-RUN ./bin/spc build --build-embed --enable-zts "$PHP_EXTENSIONS"
+RUN ./bin/spc build --build-embed --enable-zts --disable-opcache-jit "$PHP_EXTENSIONS" --with-libs="$PHP_EXTENSION_LIBS"
```

Comment:
Is it mandatory to disable the JIT?

Reply:
Not necessarily, I haven't figured out what causes the relocation problems, but compiling FrankenPHP as non-PIE fixes the issue and JIT doesn't have to be disabled.

Reply:
Unfortunately I can't really figure out what causes or fixes these issues and google isn't of much help either. All the suggestions (such as upgrading bintools) don't appear to make a difference.